docs/docs/configuration/skipping.md

```diff
@@ -114,3 +114,5 @@ A file pattern contains the analyzer it is used for, and the pattern itself, joi
 ```
 
 The prefixes are listed [here](https://github.com/aquasecurity/trivy/tree/{{ git.commit }}/pkg/fanal/analyzer/const.go)
+
+[^1]: Only for licenses not obtained from packages (additional licenses obtained using the [license-full](../scanner/license.md) flag)
```

Comment:
Can we simplify the footnote?
```suggestion

[^1]: Only work with the [license-full](../scanner/license.md) flag)
```

Reply:
I thought about this, but `license-full` flag includes licenses from packages.
Do you think this won't be confusing?

Reply:
I think it is ok. Let's start with a simple explanation. If it confuses users, we can update it.

Reply:
Updated footnote in https://github.com/aquasecurity/trivy/pull/5484/commits/b56d24161fc5bc9349db05061bd70aa8014dae8a

Reply:
Thanks!